packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -83,6 +83,7 @@ export const _IN_LOBBY_ = 'IN_LOBBY';
 export const _IN_MEETING_ = 'IN_MEETING';
 export const _INCOMING_ = 'INCOMING';
 export const _IN_ = 'IN';
+export const _S_LINE = 's=-';
```

Comment:
i think we've been trying to alphabetize the constants, can't enforce it so whatever, but might be good to shift it down